src/pages/home/search/searchModal.module.css

```diff
@@ -0,0 +1,115 @@
+.bait_input {
+  width: 100%;
+  height: 3rem;
+  border-radius: 9999px;
+  background-color: var(--secondary);
+  border: 1px solid var(--border);
+  display: flex;
+  align-items: center;
+  padding: 0.25em 1.25em;
+  color: inherit;
+  padding-left: 1em;
+  display: flex;
+  align-items: center;
+  gap: 0.5em;
+  cursor: pointer;
+  justify-content: space-between;
+  opacity: 70%;
+}
+.keyBoardShortcut {
+  font-weight: 600;
+}
+.text {
+  margin-left: -8rem;
+}
+.dialog{
+    position: fixed;
+    inset: 0px;
+    z-index: 50;
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+    backdrop-filter: blur(8px);
+}
+.overlay{
+    position: fixed;
+    inset: 0px;
+    background-color: black;
+    opacity: 0.7;
+}
+.dialog_panel{
+    position: relative;
+    inset: 0px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width:100vw;
+    max-width: 42rem; 
+    border-radius: 0.5rem;
```

Comment:
i think if you move this border-radius to form_input, border on input will look more smooth  
<img width="65" alt="Screenshot 2023-10-19 at 3 37 45 PM" src="https://github.com/chingu-voyages/v46-tier2-team-14/assets/111369805/9517859e-99ec-40b9-96dd-497dd8f92629">

Reply:
done